build/gulp/renovation-npm/index.js

```diff
@@ -155,6 +156,7 @@ addCompilationTask({
     switches: {
         installPackages: true,
     },
+    components: ['Button'],
```

Comment:
what if we use glob-style patterns ([minimatch](https://www.npmjs.com/package/minimatch)) here?

Reply:
done